src/index.js

```diff
@@ -1,7 +1,13 @@
 window.onload = function() {
 	var ajax = new XMLHttpRequest();
 	ajax.open("GET", "banner.html");
-	ajax.responseType = "document";
```

Comment:
Specifying the `responseType` to be a document causes you to get a `document` object back. This sounds nice, but there's no easy way to inject a `document` object into an existing DOM node. By removing this line, you now get a text response, which is easy to inject by setting the `innerHTML` of a node to it.